ajax/libs/jquery.ime/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "jquery.ime",
+  "version": "0.2.0",
+  "description": "jQuery based internationalization library",
+  "keywords": [
+    "internationalization",
+    "localization",
+    "ime",
+    "jquery",
+    "l10n"
+  ],
+  "author": {
+    "name": "Santhosh Thottingal",
+    "email": "santhosh.thottingal@gmail.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/wikimedia/jquery.ime.git"
+  },
+  "filename": "jquery.ime.min.js",
```

Comment:
Let's move this line in second order.
To help people see it at the first glance.
Thank you.